app/src/models/banner.ts

```diff
@@ -33,6 +33,6 @@ export type Banner =
       readonly type: BannerType.RebaseConflictsFound
       /** name of the branch that was used to rebase */
       readonly targetBranch: string
-      /** popup to be shown when opening the banner */
-      readonly popup: Popup
+      /** callback to run when user clicks on link in banner text */
+      readonly onOpenDialog: () => void
```

Comment:
This change is required because in #7117 I no longer have a `popup` for the rebase conflicts - this is moved inside a "flow" that maintains it's own state. For the moment, this callback just wraps the current behaviour.